server/lobbyconnection.py

```diff
@@ -953,6 +954,8 @@ async def launch_game(
             # options are. Currently, options for ladder are hardcoded into the
             # client.
             "name": game.name,
+            "game_type": GameType.to_string(game.game_type),
```

Comment:
Could we take the opportunity to refactor this `to_string` stuff? `Enum`s have some nice support for this already with the `__members__` dictionary and being able to simply call the constructor with the enum value. See the `Faction` enum in `server/factions.py`

```python3
>>> from enum import Enum
>>> class Foo(Enum):
...     bar = 1
...     baz = 2
... 
>>> Foo(1)
<Foo.bar: 1>
>>> Foo(1).name
'bar'
>>> Foo(1).value
1
>>> Foo.__members__
mappingproxy({'bar': <Foo.bar: 1>, 'baz': <Foo.baz: 2>})
```

Reply:
Actually it turns out I did this already in #753. So just rebase again and the changes will be there.